### PR TITLE
fix: Create storage volume before writing

### PR DESCRIPTION
**Pull request by Darkheir**

### CODE REVIEW

File: sekoia_automation/storage.py [hunk 1]
```diff
@@ -20,7 +20,7 @@
 @retry(
     reraise=True,
     wait=wait_exponential(max=6),
-    stop=stop_after_attempt(10),
+    stop=stop_after_attempt(1),
```
Comment (Darkheir) [inline sekoia_automation/storage.py:23]:
```suggestion
    stop=stop_after_attempt(10),
```

Comment (otetard) [inline sekoia_automation/storage.py]: Is that change intentional?

Comment (Darkheir) [reply sekoia_automation/storage.py]: Too fast, already fixed :-)

**Review by otetard — APPROVED** [review]
